app/services/course_search_service.rb

```diff
@@ -51,44 +52,64 @@ def call
 
 private
 
-  def sites_with_distance_from_origin
+  def distance_with_university_area_adjustment
+    university_provider_type = Provider.provider_types[:university]
+    university_location_area_radius = 10
+    <<~EOSQL.gsub(/^[\s\t]*/, "").gsub(/[\s\t]*\n/, " ").strip
```

Comment:
Very confusing. How about:

```suggestion
    <<~EOSQL.gsub(/\s+/m, " ").strip
```

or without the regexp:

```suggestion
    <<~EOSQL.tr_s(" \t\n", " ").strip
```